vars/managerPipeline.groovy

```diff
@@ -232,12 +235,14 @@ def call(Map pipelineParams) {
 
                                         if [[ ! -z "${params.scylla_ami_id}" ]] ; then
                                             export SCT_AMI_ID_DB_SCYLLA="${params.scylla_ami_id}"
+                                        elif [[ ! -z "${params.gce_image_db}" ]] ; then
+                                            export SCT_GCE_IMAGE_DB="${params.gce_image_db}"  #TODO: remove it once scylla_version supports gce image detection
                                         elif [[ ! -z "${params.scylla_version}" ]] ; then
                                             export SCT_SCYLLA_VERSION="${params.scylla_version}"
                                         elif [[ ! -z "${params.scylla_repo}" ]] ; then
                                             export SCT_SCYLLA_REPO="${params.scylla_repo}"
                                         else
-                                            echo "need to choose one of SCT_AMI_ID_DB_SCYLLA | SCT_SCYLLA_VERSION | SCT_SCYLLA_REPO"
+                                            echo "need to choose one of SCT_AMI_ID_DB_SCYLLA | SCT_GCE_IMAGE_DB | SCT_SCYLLA_VERSION | SCT_SCYLLA_REPO"
```

Comment:
in this case, it will have both `SCT_GCE_IMAGE_DB` and `SCT_SCYLLA_REPO`

Reply:
We will not, due to the `elif`. notice the difference between SCT_SCYLLA_REPO and scylla_repo_loader